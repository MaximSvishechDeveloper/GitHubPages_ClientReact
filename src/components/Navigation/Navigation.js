import React from "react";
import "./Navigation.css";
import AccountMenu from "../AccountMenu/AccountMenu";

const Navigation = () => {
  return (
    <nav className="main-nav mr3 mt3">
      <AccountMenu />
    </nav>
  );
};

export default Navigation;
