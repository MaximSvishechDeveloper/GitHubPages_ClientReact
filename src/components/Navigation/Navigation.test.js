import { shallow } from "enzyme";
import Navigation from "./Navigation";
import React from "react";

it("renders without crashing", () => {
  expect(shallow(<Navigation />)).toMatchSnapshot();
});
