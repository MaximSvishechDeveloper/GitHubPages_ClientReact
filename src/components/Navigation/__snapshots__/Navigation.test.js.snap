// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<nav
  className="main-nav mr3 mt3"
>
  <AccountMenu />
</nav>
`;
